Cargo.toml

```diff
@@ -345,6 +345,8 @@ default = [
 all-arch = ["wasmtime/all-arch"]
 winch = ["wasmtime/winch"]
 wmemcheck = ["wasmtime/wmemcheck"]
+wasi-nn-onnx = ["wasmtime-wasi-nn/onnx"] # enable ONNX backend for WASI-NN
+wasi-nn-winml = ["wasmtime-wasi-nn/winml"] # enable WinML backend for WASI-NN
```

Comment:
I don't think we need to add this top-level feature because we have workspaces. We should be able to reach down and control sub-crate features by doing: `cargo build --features wasmtime-wasi-nn/onnx`, e.g.